src/airport.js

```diff
@@ -4,8 +4,6 @@ class Airport {
 
   slotsOccupied = [];
 
-  slotsAvailable = this.slotsOccupied.length = 10;
```

Comment:
Was trying different ways to set a default slot length, I ended up doing some reading on the MDN website specifically around working with arrays. This did not initially solve my issue but I worked it out later on.